AnimalsTest/IBigCats.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
I bet not all of these usings are needed. Consider enabling editorconfig rule which errors out unnecessary usings.

Reply:
This applies to all the files